StorageUtility/StorageUtility.pro.user

```diff
@@ -0,0 +1,214 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file used for?
The extension seems like to reflect user configuration. It may cause to confliction when working with team. Please remove this.